docs/source/notes/get_start_xpu.rst

```diff
@@ -0,0 +1,419 @@
+
+Pytorch 2.4: Getting Started on Intel GPU
+=========================================
+
+The support for Intel GPUs is released alongside PyTorch v2.4.
+
+This release only supports build from source for Intel GPUs.
+
+Hardware Prerequiste
+--------------------
+
+.. list-table::
+   :header-rows: 1
+
+   * - Supported Hardware
+     - Intel® Data Center GPU Max Series
+     - Intel® Core™ Ultra Processors
```

Comment:
Are Ultra Processors CPU-only?

Reply:
remove Ultra Processors since we don't support in this release